source/getting-started-edge/index.rst

```diff
@@ -89,6 +89,142 @@ resource usage. The dashboard looks  something like this:
 
    An overview of your project's current resource usage
 
+Setting up CLI
+--------------
+
+Right now, edge device reservations can only be made through the command line 
+interface. This means that you must follow the instructions
+`here <../technical/cli.html>`_ to set up your blazar client. If you have
+previously installed blazar, you will need to reinstall in order to add in the
+new changes for edge devices
+
+.. code-block:: shell
+
+  pip install git+https://github.com/chameleoncloud/blazar@chameleoncloud/train
+
+Be sure to use the the OpenStack RC file downloaded from the Edge site, which
+means you should be logged into the GUI at
+`CHI@Edge <https://chi.edge.chameleoncloud.org>`_. Once there, you can follow
+the `same instructions <../technical/cli.html#the-openstack-rc-script>`_
+as is done on the other sites to download this file.
+
+The Lease Calendar
+------------------
+
+Visit the calendar for edge devices by going to `this link <https://chi.edge.chameleoncloud.org/project/leases/device_calendar/>`_.
+This calendar lets you discover when resources are available to use. The *Y* 
+axis of this chart represents the different edge devices in the system, and the
+*X* axis represents time.
+
+.. figure:: device_calendar.png
+   :alt: The edge device calendar
+   :figclass: screenshot
+
+   A Gantt chart displaying when devices are available.
```

Comment:
Can we just call it a chart, or a bar chart? It's not really a gantt chart, we just abused some library normally intended for gantt viz :D